commit b3159470aa8f6a5dd8227a8be557e23018884d4b
Author: Romain Izard <romain.izard.pro@gmail.com>
Date:   Tue Nov 20 17:57:37 2018 +0100

    ARM: dts: at91: sama5d2: use the divided clock for SMC
    
    [ Upstream commit 4ab7ca092c3c7ac8b16aa28eba723a8868f82f14 ]
    
    The SAMA5D2 is different from SAMA5D3 and SAMA5D4, as there are two
    different clocks for the peripherals in the SoC. The Static Memory
    controller is connected to the divided master clock.
    
    Unfortunately, the device tree does not correctly show this and uses the
    master clock directly. This clock is then used by the code for the NAND
    controller to calculate the timings for the controller, and we end up with
    slow NAND Flash access.
    
    Fix the device tree, and the performance of Flash access is improved.
    
    Signed-off-by: Romain Izard <romain.izard.pro@gmail.com>
    Signed-off-by: Alexandre Belloni <alexandre.belloni@bootlin.com>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm/boot/dts/sama5d2.dtsi b/arch/arm/boot/dts/sama5d2.dtsi
index 61f68e5c48e9..b405992eb601 100644
--- a/arch/arm/boot/dts/sama5d2.dtsi
+++ b/arch/arm/boot/dts/sama5d2.dtsi
@@ -308,7 +308,7 @@
 				  0x1 0x0 0x60000000 0x10000000
 				  0x2 0x0 0x70000000 0x10000000
 				  0x3 0x0 0x80000000 0x10000000>;
-			clocks = <&mck>;
+			clocks = <&h32ck>;
 			status = "disabled";
 
 			nand_controller: nand-controller {
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1068_b3159470aa8f6a5dd8227a8be557e23018884d4b.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1068_b3159470aa8f6a5dd8227a8be557e23018884d4b.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
